tests/generate-mutation-query.js

```diff
@@ -14,6 +16,9 @@ const moveData = {
 				project: {
 					name: project,
 					id: 'MDc6UHJvamVjdDQwNzU5MDI='
+				},
+				column: {
+					name: 'Hotbox'
```

Comment:
I could use some help here. I am unsure of this is correct. I've been running on the assumption the data is laid out like this. It shouldn't be too hard to change if it's different. Essentially I just need to be able to get the column the card is currently in.